xchange-bitcointoyou/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.knowm.xchange</groupId>
+		<artifactId>xchange-parent</artifactId>
+		<version>4.3.3-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>xchange-bitcointoyou</artifactId>
+
+	<name>XChange BitcoinToYou</name>
+	<description>XChange implementation for the BitcoinToYou Exchange</description>
+
+	<url>http://xeiam.com</url>
+	<inceptionYear>2018</inceptionYear>
+
+	<organization>
+		<name>Xeiam, LLC</name>
+		<url>http://xeiam.com</url>
+	</organization>
+
+	<!-- Parent provides default configuration for dependencies -->
+	<dependencies>
+	
+		<dependency>
+			<groupId>${project.groupId}</groupId>
+			<artifactId>xchange-core</artifactId>
+			<version>${project.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.java-websocket</groupId>
+					<artifactId>Java-WebSocket</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+
+		<dependency>
```

Comment:
Isn't pusher for streaming? XChange doesn't support streaming - only REST.

Reply:
Yeah, unuseful dependency. Removed!